### DISCUSSION TITLE
Sqlite

### DISCUSSION THREAD

**Discussion by iezed** [nickjj/docker-flask-example#20] [Q&A]
Hi, can I use this template without Redis but with SQLite instead of PostgreSQL?

**Comment by nickjj** [comment]
Sure, SQLAlchemy supports multiple database backends. You can connect to `sqlite:///myapp.db` instead of Postgres.

You can also remove Postgres and Redis from `docker-compose.yml`. You'd want to adjust the `up` Blueprint to not connect to Redis too.

**Reply by iezed** [reply]
Thanks.
